code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java

```diff
@@ -373,4 +377,65 @@ private Map<String, String> cleanContextData(Map<String, Object> eventData) {
 		}
 		return cleanedData;
 	}
+
+	/**
+	 * Adds various Analytics standard properties into the final track event payload. Creates required
+	 * parts of the hierarchy if they are not already present in the original event data. This should
+	 * be used only after first validating the event data.
```

Comment:
Adds the following keys to the given data map:
* __adobe.analytics.cp 
* __adobe.analytics.contextData.a.AppId

Creates required paths if they are not already present in the original data map. This should be used only after first validating the track event has valid data from the public API.